go/cmd/aggregate-crls/aggregate-crls.go

```diff
@@ -116,6 +116,63 @@ func (ae *AggregateEngine) findCrlWorker(ctx context.Context, wg *sync.WaitGroup
 	resultChan <- issuerCrls
 }
 
+func (ae *AggregateEngine) crlFetchWorkerProcessOne(ctx context.Context, crlUrl url.URL, issuer storage.Issuer) (string, error) {
+	err := os.MkdirAll(filepath.Join(*crlpath, issuer.ID()), permModeDir)
+	if err != nil {
+		glog.Warningf("Couldn't make directory: %s", err)
+		return "", err
+	}
+
+	filename := makeFilenameFromUrl(crlUrl)
+	tmpPath := filepath.Join(*crlpath, issuer.ID(), filename+".tmp")
+	finalPath := filepath.Join(*crlpath, issuer.ID(), filename)
+
+	err = downloader.DownloadFileSync(ctx, ae.display, crlUrl, tmpPath, 3)
+	if err != nil {
+		glog.Warningf("[%s] Could not download %s to %s: %s", issuer.ID(), crlUrl.String(),
+			tmpPath, err)
```

Comment:
Is it guaranteed that there is no file at `tmpPath` here? (the other branches clean up the temporary file, so I was wondering if we wanted to do that here as well)

Reply:
Good call. Cleaning it up.